### PR TITLE
Feature: adds background colour to request options

### LINKED ISSUE #151: Accessibility issue: Support color blindness in V2.
GE as it is has accessibility issues to support colorblind people, we have a meeting scheduled next week to speak with accessibility engineers on how to implement this for Graph Explorer.

### LINKED ISSUE #230: Add background color on the Request Options tab
**Describe the solution you'd like**
Add background color on the Request Options tab next to the Query tab like in the screenshot below: 

image

### PR DESCRIPTION
**Pull request by cwahome**
## Overview

Adds background color on the Request Options tab.
Helps with accessibility

Closes #151 
Closes #230 

### Demo

Get
image

Post
image

Put
image

Patch
image

Delete
image

### CODE REVIEW

**Comment by jobala** [comment]
I remember Linda raised an issue around color blindness which I think was related to these http verb colors. Do these solve that? If they do then this PR also closes #151

File: src/app/views/query-runner/QueryInput.tsx [hunk 1]
```diff
@@ -54,7 +55,16 @@ export class QueryInput extends Component<IQueryInputProps, any> {
             selectedKey={selectedVerb}
             options={httpMethods}
             disabled={mode === Mode.TryIt}
-            styles={{ title: { paddingRight: 0 } }}
+            styles={{
```
Comment (jobala) [inline src/app/views/query-runner/QueryInput.tsx]: Move inline styles to a `.styles.ts` file

**Comment by cwahome** [comment]
> I remember Linda raised an issue around color blindness which I think was related to these http verb colors. Do these solve that? If they do then this PR also closes #151

Yeah. They do... Editing the description to add that issue

**Comment by bettirosengugi** [comment]
> I remember Linda raised an issue around color blindness which I think was related to these http verb colors. Do these solve that? If they do then this PR also closes #151

I think you can test using the tool I shared with you just to be sure @jobala

**Comment by jobala** [comment]
@bettirosengugi I never got the tool to run.

**Pull request merged**